tests/direct-integrator-tests.cpp

```diff
@@ -37,12 +40,23 @@ int main() {
 	std::shared_ptr<Material> whiteMatte(new MatteMaterial(whiteTexture));
 	std::shared_ptr<EmissionMaterial> emissionMat(new EmissionMaterial(lightTexture, 1.3f));
 
+	// Suggestion: Assign all parameters in single variable
+	// Eg what does the first 500 mean for options?
```

Comment:
That's kinda fine. We get these problems all the time. Every function has values that you usually don't know what they refer to. Of course this can be done better in this specific situation but overall it's usually fine